File: Check if a String Is an Acronym of Words.go
package main

func isAcronym(words []string, s string) bool {
	acronym := ""
	for i := 0; i < len(words); i++ {
		acronym += string(words[i][0])
	}
	if acronym == s {
		return true
	}
	return false
}

File: Check if a String Is an Acronym of Words.js
function isAcronym(words, s){
    //empty string to store acronym
    let acronym = "";
    //for loop to get the acronym
    for (let i = 0; i < words.length; i++){
        acronym += words[i][0];
    }
    //compare acronym with s string
    if (acronym === s){
        return true;
    }
    return false;
}
